daemon/volumes.go

```diff
@@ -385,15 +385,14 @@ func copyExistingContents(source, destination string) error {
 // copyOwnership copies the permissions and uid:gid of the source file
 // into the destination file
 func copyOwnership(source, destination string) error {
-	var stat syscall.Stat_t
-
-	if err := syscall.Stat(source, &stat); err != nil {
+	stat, err := system.GetStat(source)
```

Comment:
This wouldn't compile on windows because there's no Stat_t type on windows and what's returned from os.Stat().Sys doesnt have fields like Uid/Gid, because it's windows. We extracted stat functionality in early days of windows port into a separate package so that we can satisfy linux/windows implementations through a common interface.